### PR TITLE
Add Educational Games

### PR DESCRIPTION
**Pull request by vinkla**
https://github.com/vinkla/awesome-edutainment

A curated list of awesome gamification to learn editors, languages, programming, etc.
- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. _(For example: `Add Node.js`)_
- The list I added
  - [x] **has been around for at least 20 days,**
  - [x] is a non-generated Markdown file in a GitHub repo,
  - [x] is not a duplicate,
  - [x] only has awesome items with descriptions _(Awesome lists are curations of the best, not of everything)_,
  - [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
  - [x] has a Table of Contents section named `Contents` as the first section,
  - [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) _(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)_,
  - [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
  - [x] has been checked for inconsistent formatting, spelling, and grammar _(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)_,
  - [x] has been added at the bottom of the appropriate category.

### CODE REVIEW

**Comment by vinkla** [comment]
Haven't added the awesome badge yet. Want to make sure you are okay with it first.

**Comment by davisonio** [comment]
Nice list, only one point not followed:

> has a Table of Contents section named **Contents** as the first section,

**Comment by vinkla** [comment]
So rename Table of contents to Contents?

**Comment by vinkla** [comment]
@davisonio all done!

**Comment by sindresorhus** [comment]
- The contributing.md have just been lazily copy-pasted without adapting the content to your list.
- Would be nice to have more non-programming gamification resources. There are for example lots of great ones regarding exercise.
- The artwork on the top right, while nice, is a bit too big.
- `Contents` should be a `##` section.
- `ReturnTrue - Return true to win.` Not very useful description.
- I would like to see more than just services/games. Include some blog posts, tutorials, videos of talks, book, learning material, etc.
- Use the [SVG version](http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg) of the CC0 license badge.
- `Try Git - Learn how to use Git with Code School’s interactive course, Try Git.` No need to mention `Try Git` at the end.
- Most descriptions could be improved.
- No point in including the Travis build status badge. It has no value here.

File: readme.md [hunk 1]
```diff
@@ -347,7 +347,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Dive into Machine Learning](https://github.com/hangtwenty/dive-into-machine-learning)
 - [Computer History](https://github.com/watson/awesome-computer-history)
 - [Programming for Kids](https://github.com/HollyAdele/awesome-programming-for-kids)
-
```
Comment (sindresorhus) [inline readme.md]: Don't remove the newline.

File: readme.md [hunk 2]
```diff
@@ -347,7 +347,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
...
+- [Gamification](https://github.com/vinkla/awesome-gamification)
```
Comment (sindresorhus) [inline readme.md]: Add a succinct description.

Comment (vinkla) [inline readme.md]: What does succinct mean?

Comment (sindresorhus) [inline readme.md]: In the time you used to write the question you could have easily Google'd it.

**Comment by vinkla** [comment]
Thanks for you feedback! I've tried to fix most of the issues based on your feedback.

> I would like to see more than just services/games. Include some blog posts, tutorials, videos of talks, book, learning material, etc.

I agree. Would be nice nice to include more topics. Any ideas are welcome!

> I would like to see more than just services/games. Include some blog posts, tutorials, videos of talks, book, learning material, etc.

Any ideas or links are welcome.

File: readme.md (hunk 2 above)
Comment (vinkla) [inline readme.md]: Fixed.

**Comment by sindresorhus** [comment]
ping

File: readme.md (hunk 1 above)
Comment (vinkla) [inline readme.md]: Fixed.

**Comment by vinkla** [comment]
Renamed the list to edutainment https://github.com/vinkla/awesome-edutainment/issues/2

**Comment by sindresorhus** [comment]
> Like awesome-educational-games? Reach out to...

Would sounds better like:

> Like this list? Reach out to...

---

> Any ideas or links are welcome.

Sure, but it's your job as a list creator to create a great initial version. Maybe add https://education.minecraft.net/

**Comment by vinkla** [comment]
Thanks!

**Pull request merged**